packages/contracts-bedrock/src/L2/L2ToL2CrossDomainMessenger.sol

```diff
@@ -61,8 +64,8 @@ contract L2ToL2CrossDomainMessenger is ISemver, TransientReentrancyAware {
     uint16 public constant messageVersion = uint16(0);
 
     /// @notice Semantic version.
-    /// @custom:semver 1.0.0-beta.16
-    string public constant version = "1.0.0-beta.16";
```

Comment:
In a separate PR, I would suggest taking a pass for any other contracts that still have the legacy `-beta*` qualifiers and remove those similar to what we did here